### ISSUE TITLE
[Bug] [UWP] Setting Translation/Rotation/Scale to all zero on UWP does not reset transformation

### ISSUE THREAD

**Issue by activa** [xamarin/Xamarin.Forms#7385]
### Description

When changing any of the transformations on a VisualElement (translation/rotation/scale) and then resetting it back to zero does nothing on UWP. The last property that is reset to zero is ignored.

This is only an issue on UWP.

### Steps to Reproduce

1. Set TranslationX to non-zero
2. Set TranslationX to zero

### Expected Behavior

Element is moved back to the origin

### Actual Behavior

Element stays at the translated position from step 1

### Basic Information

- Version with issue: all
- Last known good version: N/A
- Platform Target Frameworks:
  - UWP: all versions

### Screenshots

After setting translation to 30,20:
![ApplicationFrameHost_2019-09-04_15-39-54](https://user-images.githubusercontent.com/650710/64290624-0a4f7180-cf2c-11e9-9b5a-5859148b2e53.png)

After setting TranslationX to 0:
![ApplicationFrameHost_2019-09-04_15-40-27](https://user-images.githubusercontent.com/650710/64290651-189d8d80-cf2c-11e9-92f3-0f2477d08140.png)

After setting TranslationY to 0:
![ApplicationFrameHost_2019-09-04_15-40-41](https://user-images.githubusercontent.com/650710/64290669-23582280-cf2c-11e9-9344-df5ce6a51c0a.png)

The last step does nothing. The box is not moved to position Y=0

### Reproduction Link

Repro case:

```csharp
public class IssueXXXX : TestContentPage
{
    View _box;

    protected override void Init()
    {
        _box = new BoxView
        {
            WidthRequest = 50,
            HeightRequest = 50,
            HorizontalOptions = LayoutOptions.Start,
            VerticalOptions = LayoutOptions.Start,
            BackgroundColor = Color.Red
        };

        var button1 = new Button { Text = "Set TranslationX/Y to 30,20", Command = new Command(() => MoveBox(30, 20)) };
        var button2 = new Button { Text = "Set TranslationX to 0", Command = new Command(() => MoveBox(x: 0)) };
        var button3 = new Button { Text = "Set TranslationY to 0", Command = new Command(() => MoveBox(y: 0)) };

        Content = new StackLayout { Children = { new StackLayout { HeightRequest = 200, Children = { _box } }, new StackLayout { Orientation = StackOrientation.Horizontal, Children = { button1, button2, button3 } } } };
    }

    private void MoveBox(double? x = null, double? y = null)
    {
        if (x != null)
            _box.TranslationX = x.Value;

        if (y != null)
            _box.TranslationY = y.Value;
    }
}
```

**Comment by activa** [comment]
Can this be reviewed please?

**Comment by samhouts** [comment]
closed by #7386
